Client/package.json

```diff
@@ -11,12 +11,13 @@
 		"preview": "vite preview"
 	},
 	"dependencies": {
-		"@emotion/react": "^11.11.4",
-		"@emotion/styled": "^11.11.5",
+		"6": "^0.0.1",
```

Comment:
@ajhollid I figured out what happened here. At some point I upgraded mui to test some ways of dealing with theming, and I may have installed mui 6 instead of mui@6 xD

Solved here https://github.com/bluewave-labs/bluewave-uptime/pull/1152/commits/7eaba0a47d19bc39e34933ff0ddcface1d4ad966